code/modules/spacepods/spacepod.dm

```diff
@@ -73,7 +73,7 @@
 	pr_int_temp_processor = new /datum/global_iterator/pod_preserve_temp(list(src))
 	pr_give_air = new /datum/global_iterator/pod_tank_give_air(list(src))
 	equipment_system = new(src)
-	equipment_system.installed_modules = new/list()
+	equipment_system.installed_modules = list()
```

Comment:
`list()` returns a new, empty list; this is already used in over 2200 places in the codebase. `list(0)` should return a list with a single 0 inside of it, which is probably undesirable in most cases.